drivers/unix/ip_unix.cpp

```diff
@@ -38,19 +38,7 @@
   #include <windows.h>
   #include <stdio.h>
   #ifndef UWP_ENABLED
-    #if defined(__MINGW32__ ) && (!defined(__MINGW64_VERSION_MAJOR) || __MINGW64_VERSION_MAJOR < 4)
-      // MinGW-w64 on Ubuntu 12.04 (our Travis build env) has bugs in this code where
```

Comment:
It's just that MinGW-w64 before 4.0 is broken beyond repair, I've tried including myself the missing headers but there are recursive dependencies between all headers and I couldn't find a way to make it build.